examples/web_agent/env.example

```diff
@@ -0,0 +1,6 @@
+# Copy this file to .env and set the appropriate environment variable(s) for your application.
+AIRTOP_API_KEY=""
+requires_login="False"
+TARGET_URL="https://en.wikipedia.org/wiki/A.I._Artificial_Intelligence"
+DB_NAME="homolog.db"
```

Comment:
might want to add to .gitignore

Reply:
the .env is in gitignore, sorry, I should add what?

Reply:
i meant `homolog.db`